scripts/postinstall.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const commitlintrcPath = `${process.env.INIT_CWD}/.commitlintrc.js`;
+
+fs.open(commitlintrcPath, 'wx', err => {

```

Comment:
@linusklingzell No need to be asynchronous in this file, this is gets run in a synchronous process anyways.